easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.84.0-GCCcore-14.2.0.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'MesonNinja'
+
+name = 'GObject-Introspection'
+version = '1.84.0'
+
+homepage = 'https://gi.readthedocs.io/en/latest/'
+description = """GObject introspection is a middleware layer between C libraries
+ (using GObject) and language bindings. The C library can be scanned at
+ compile time and generate a metadata file, in addition to the actual
+ native C library. Then at runtime, language bindings can read this
+ metadata and automatically provide bindings to call into the C library."""
+
+toolchain = {'name': 'GCCcore', 'version': '14.2.0'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+patches = ['GObject-Introspection-1.80.1_install-GLib-GIR-files.patch']
+checksums = [
+    {'gobject-introspection-1.84.0.tar.xz': '945b57da7ec262e5c266b89e091d14be800cc424277d82a02872b7d794a84779'},
+    {'GObject-Introspection-1.80.1_install-GLib-GIR-files.patch':
+     'c1909f1b7fd30784ae789ac0b5e45e0ca3dd2456890b864efa86a2f8c2e563aa'},
+]
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('pkgconf', '2.3.0'),
+    ('CMake', '3.31.3'),
+    ('Meson', '1.6.1'),
+    ('Ninja', '1.12.1'),
+    ('Bison', '3.8.2'),
+    ('cairo', '1.18.4'),
+    ('flex', '2.6.4'),
+]
+
+dependencies = [
+    ('Python', '3.13.1'),
+    ('GLib', '2.85.1'),
+    ('libffi', '3.4.5'),
+    ('util-linux', '2.41'),
+]
+
+preconfigopts = "env GI_SCANNER_DISABLE_CACHE=true "
```

Comment:
Do you rally want `env` here and not just "GI_SCANNER_DISABLE_CACHE=true "
Comment